README.md

```diff
@@ -6,13 +6,28 @@ This is my educational project within IT KPI Python mentorship program by @webkn
 
 ## Installation
 
-Soon
+1. Clone this repository:
+```console
+git clone https://github.com/kpi-web-guild/django-girls-blog-Martolivna.git
```

Comment:
Using the `console` lexer, the command lines should be prefixed with a prompt, which is usually `$ `. The non-prompt lines are considered command output.
```suggestion
$ git clone https://github.com/kpi-web-guild/django-girls-blog-Martolivna.git
```
For example:
```console
$ echo ${HOME}
/home/username

$ echo hey
hey
```
This lexer can apply some highlighting to the prompt-lines and will leave the output untouched.

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/6791bbf4a38719468006c4991bf5b12ef6cfb08d

Reply:
Marking this thread as resolved.